.github/workflows/cli-binary-release.yaml

```diff
@@ -0,0 +1,66 @@
+name: Build and release wgc binaries
```

Comment:
As discussed: It would be awesome, if we could generate wgc previews for customers.

Reply:
https://github.com/wundergraph/cosmo/actions/runs/13503955404?pr=1620